tracex_project/db_results/views.py

```diff
@@ -205,100 +222,104 @@ def create_bar_chart(data, x_title, y_title):
 
 
 class EvaluationView(FormView):
-    """View for displaying all extracted traces and DFG image with filter selection."""
+    """View for displaying all extracted traces and DFG images with filter selection options."""
```

Comment:
This would technically be more correct
```suggestion
    """View for displaying all extracted traces and their corresponding DFG image with filter selection options."""
```